### DISCUSSION TITLE
Stirling-pdf on macos?

### DISCUSSION THREAD

**Discussion by houdini69** [Stirling-Tools/Stirling-PDF#1531] [General]
Has anyone already installed stirling-pdf on macos? I make it work on Debian with Docker without problem but I also have a Mac M2 at work and I would like to install it there too.

**Comment by shawnzxx** [comment]
I also have issues installing on my Mac using docker-compose. Currently, I’m using native Java to install Stirling-pdf on my M4 Mac.

**Comment by Frooodle** [comment]
We tested it fine on MacOS previously we also actually have a mac installer (non-docker) for the ultra-lite version in our releases

**Comment by Frooodle** [comment]
What mac issues do you see?

**Comment by y9c** [comment]
<img width="254" alt="image" src="https://github.com/user-attachments/assets/59cf88e9-e20e-474b-8561-5242af27e5d9" />

**Reply by Frooodle** [reply]
Which file did you use for this?

**Reply by y9c** [reply]
<img width="536" alt="image" src="https://github.com/user-attachments/assets/43fb4d90-776f-4965-8b3a-5e5cff874b98" />

**Reply by y9c** [reply]
https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.44.3/Stirling-PDF-mac-installer.dmg

**Reply by DarioGii** [reply]
@y9c what version of MacOS are you on?

**Reply by y9c** [reply]
The latest one but on Intel chip

**Reply by Runskey** [reply]
The same problem.

**Reply by Hanlin-its** [reply]
The same problem.🧐

**Reply by Frooodle** [reply]
For intel chips we have a new intel installer which should help 
https://objects.githubusercontent.com/github-production-release-asset-2e65be/594155488/299aa713-25f4-4e71-bb18-41b108c6633a?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20250411%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250411T074721Z&X-Amz-Expires=300&X-Amz-Signature=efb8d0095f94bf6c00e73f8b148d03233e8b51927aee45f8ed84034be0595aad&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3DStirling-PDF-mac-x86_64-installer.dmg&response-content-type=application%2Foctet-stream

**Reply by Hanlin-its** [reply]
<img width="473" alt="截圖 2025-04-11 下午3 54 13" src="https://github.com/user-attachments/assets/316219d4-64d6-415f-95c0-1469d45bb422" />
<img width="466" alt="截圖 2025-04-11 下午3 57 19" src="https://github.com/user-attachments/assets/665cbc92-bc7b-4ca3-a17d-054395b3b10b" />

I tried the installation file provided above, but unfortunately it still doesn't work. Provide my current computer information

**Reply by DarioGii** [reply]
We've just updated the Intel version, hopefully it should work now

**Comment by DarioGii** [comment]
We've released a new version of the app for Intel chips in this [0.45.5](https://github.com/Stirling-Tools/Stirling-PDF/releases/tag/v0.45.5) update. This should resolve the issues people were seeing with previous installations. You can download it [here](https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.45.5/Stirling-PDF-mac-x86_64-installer.dmg)

**Reply by y9c** [reply]
Thanks. Verison 0.45.5 is still not working. Now the program exits immediately after clicking the icon.

**Reply by DarioGii** [reply]
Could you share your logs? They should be at `/Users/<your_username>/Library/Application Support/Stirling-PDF/logs`

**Reply by y9c** [reply]
Thank you. But I think there is not logs folder for Stringling intel machine. 

```shell
ls -d "/Users/$USER/Library/Application Support/Stirling-PDF"
ls: cannot access '/Users/y9c/Library/Application Support/Stirling-PDF': No such file or directory
```

**Reply by DarioGii** [reply]
Can you try `ls -d "/Users/$USER/Library/Application Support/` and see if it's there? It will probably be Stirling-PDF (x86_64) in your case

**Reply by y9c** [reply]
Hi @DarioGii, This directory do not contain any file or folder relative to "Stringling"

**Comment by Hanlin-its** [comment]
<img width="956" alt="截圖 2025-04-21 下午2 04 59" src="https://github.com/user-attachments/assets/118b2906-eca0-4719-b0ee-8466e8305b9f" />
Thank you for your guidance. I tried downloading the latest version again, and the installation was successful. However, when I click the program icon to launch it, nothing happens. I checked Activity Monitor, and the software doesn't appear to be running there either. Additionally, I looked in the log file path you provided, but the corresponding folder doesn't seem to exist.

**Reply by DarioGii** [reply]
Hi @Hanlin-its, we have released version 0.46.0 with the fix to this issue. Can you try and install it again? You can find it [here](https://github.com/Stirling-Tools/Stirling-PDF/releases/tag/v0.46.0)

**Comment by y9c** [comment]
I found this is relative to java version. 

I alternative solution is install java and start the application in command line by:

java -jar '/Applications/Stirling-PDF (x86_64).app/Contents/app/Stirling-PDF-0.45.5.jar'

**Reply by Frooodle** [reply]
If you install java does the application work without the commandline hack?

**Reply by y9c** [reply]
No, clicking the icon still does not launch the program.

**Reply by DarioGii** [reply]
The wrong version of the JDK was being used when building the installer, thanks for highlighting this issue. It should now be resolved in the [latest release](https://github.com/Stirling-Tools/Stirling-PDF/releases/tag/v0.46.0).
